benchmarks/linear_algebra/blas/level1/copy/copy_2.cpp

```diff
@@ -0,0 +1,51 @@
+/**
```

Comment:
The name of this file is not correct. You have to name the files in a way that makes us able to run them using the compile_and_run.sh script. Can you please look at the naming convention by looking at the existing benchmarks and follow the same convention? Can you make sure your naming is correct by testing using that script?

Reply:
okay, i will

Reply:
Salem Dr. @rbaghdadi 
The implementation of wrapper functions in the tutorial and benchmarks is completely different, I needed to recreate these functions.
My classmates have implemented COPY and SYR2, that is why I recreated only SPR and the code can be run using the script.
sorry for bothering.